web/src/main/java/io/fabric8/launcher/web/api/OpenShiftResource.java

```diff
@@ -87,14 +91,15 @@ public JsonArray getSupportedOpenShiftClusters(@Context HttpServletRequest reque
                     .map(OpenShiftCluster::getId)
                     .forEach(arrayBuilder::add);
         } else {
-            IdentityProvider identityProvider = this.identityProviderInstance.get();
-            clusters.stream()
+            final Set<String> clusterIds = clusters.stream()
                     .filter(b -> !OSIO_CLUSTER_TYPE.equalsIgnoreCase(b.getType()))
                     .map(OpenShiftCluster::getId)
-                    .forEach(clusterId -> identityProvider.getIdentity(clusterId)
-                            .ifPresent(token -> arrayBuilder.add(clusterId)));
+                    .collect(Collectors.toSet());
```

Comment:
Don't change OpenShiftResource, it will be removed and should work sequentially.